pytorch_lightning/strategies/deepspeed.py

```diff
@@ -728,13 +728,20 @@ def deepspeed_engine(self):
     def _multi_device(self) -> bool:
         return self.num_processes > 1 or self.num_nodes > 1
 
-    def save_checkpoint(self, checkpoint: Dict, filepath: _PATH) -> None:
+    def save_checkpoint(self, checkpoint: Dict, filepath: _PATH, storage_options: Optional[Any] = None) -> None:
         """Save model/training states as a checkpoint file through state-dump and file-write.
 
         Args:
             checkpoint: The checkpoint state dictionary
             filepath: write-target file's path
+            storage_options: raises exception if passed in,
+                not used for ``DeepSpeedStrategy`` as ``CheckpointIO`` is not used
```

Comment:
See the docstring style here for desribing exceptions: https://github.com/PyTorchLightning/pytorch-lightning/blob/79c4e5de60685dbc895641b0139ffc6180d069aa/pytorch_lightning/utilities/finite_checks.py#L34-L43